### PR TITLE
LPS-96590 inline method

### PR DESCRIPTION
**Pull request by gamerson**

### CODE REVIEW

File: modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/internal/LiferayBaseDefaultsPlugin.java [hunk 1]
```diff
@@ -17,23 +17,77 @@
...
+			taskContainer.create(
+				SetBuildProfileTask.SET_BUILD_PROFILE_TASK_NAME,
+				SetBuildProfileTask.class);
+		}
+
+		GradleUtil.addTask(
+			project, CleanBuildProfileTask.CLEAN_BUILD_PROFILE_TASK_NAME,
+			CleanBuildProfileTask.class);
+
+		logger.lifecycle(
```
Comment (gamerson) [inline modules/sdk/gradle-plugins-defaults/src/main/java/com/liferay/gradle/plugins/defaults/internal/LiferayBaseDefaultsPlugin.java:78]: No need to log this.

**Comment by christopherbryanboyd** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
Pull request submitted to https://github.com/christopherbryanboyd/liferay-portal/pull/16. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by christopherbryanboyd** [comment]
Pull request submitted to https://github.com/gamerson/liferay-portal/pull/284. [See changes here.](https://github.com/christopherbryanboyd/liferay-portal/compare/ad94d118e85e~1...ad94d118e85e) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**